install/kubernetes/cilium/README.md

```diff
@@ -123,10 +123,14 @@ contributors across the globe, there is almost always someone available to help.
 | enableK8sEventHandover | bool | `false` | Configures the use of the KVStore to optimize Kubernetes event handling by mirroring it into the KVstore for reduced overhead in large clusters. |
 | enableXTSocketFallback | bool | `true` |  |
 | encryption.enabled | bool | `false` | Enable transparent network encryption. |
-| encryption.keyFile | string | `"keys"` | Name of the key file inside the Kubernetes secret configured via secretName. This option is only effective when encryption.type is set to ipsec. |
-| encryption.mountPath | string | `"/etc/ipsec"` | Path to mount the secret inside the Cilium pod. This option is only effective when encryption.type is set to ipsec. |
+| encryption.ipsec | object | `{"keyFile":"keys","mountPath":"/etc/ipsec","secretName":"cilium-ipsec-keys"}` | Deprecated in favor of encryption.ipsec.interface. The interface to use for encrypted traffic. This option is only effective when encryption.type is set to ipsec. interface: eth0 |
```

Comment:
Looking at the description: is there a chance there was a mixup between `encryption.ipsec` and `encryption.ipsec.interface`? I can't seem to find neither `encryption.interface` nor `encryption.ipsec.interface`, for which I believe this description would be more adequate. Seems like maybe we actually were missing documentation for `encryption.interface`? 🤔 

Regarding the `encryption.ipsec` container itself: I would probaby expect something more in line with the rest of the containers, which is either:
- a generic description (e.g. `IPSec-specific configuration`)
- or the container not even being documented

I would tend to favor this second option since we are already documenting every single subkey and also providing defaults for all of them in `values.yaml`, I feel like in this case also documenting the container itself and its default value is unneeded/redundant.

Reply:
Nice catch, I think that's because the interface value is commented as well
```
  # -- Deprecated in favor of encryption.ipsec.interface.
  # The interface to use for encrypted traffic.
  # This option is only effective when encryption.type is set to ipsec.
  # interface: eth0
```
and so the whole comment bind to the next item (`ipsec`)

> I would tend to favor this second option since we are already documenting every single subkey and also providing defaults for all of them in values.yaml, I feel like in this case also documenting the container itself and its default value is unneeded/redundant.

yep, agree :+1: (that comment was not intentional)